src/main/java/com/altenheim/kalender/controller/viewController/AnimationController.java

```diff
@@ -0,0 +1,52 @@
+package com.altenheim.kalender.controller.viewController;
+
+import com.altenheim.kalender.interfaces.IAnimationController;
+import javafx.animation.ScaleTransition;
+import javafx.scene.Node;
+import javafx.scene.shape.Circle;
+import javafx.scene.text.Text;
+import javafx.util.Duration;
+
+
+public class AnimationController implements IAnimationController 
+{
+    public AnimationController()
```

Comment:
leerer constructor ist nicht nötig

Reply:
doch weil hier noch Sachen reinkommen. Ich hab dir doch gesagt ich mache den PR damit hier nicht noch mehr unnötige Merge Konflikte reinkommen. Das ist grad Work in Progress also bitte Anmerkungen inhaltlicher Art :-)